src/makerom/rom_header.s

```diff
@@ -25,4 +30,7 @@
 #elif OOT_REGION == REGION_EU
 /* 0x3E */ REGION(PAL)
 #endif
+#else
+/* 0x3B */ .byte 0,0,0,0
```

Comment:
is `REGION(NULL)` not working here? it should be 0 right?

Reply:
I feel like if I do this for one of the bytes I should do it for all of them, which is probably ok except for `MEDIUM`, I'll think about what to do there

Reply:
in the other direction, what about zeroing out the entire rest of the header instead of leaving the padding and revision untouched? (Why did they zero out the header for iQue anyway?)

Reply:
Switched to zeroing the entire header following the checksum for now

I think a lot of the header is just zero because I think it's populated in 2 passes with 2 different tools (`makerom` and `nrdc` iirc) so they probably didn't run the latter